src/Home/Home.tsx

```diff
@@ -49,59 +75,280 @@ const Home = ({
+
+  const headerText = appIsActive
+    ? t("home.bluetooth.tracing_on_header")
+    : t("home.bluetooth.tracing_off_header")
+
+  const subheaderText = appIsActive
+    ? t("home.bluetooth.all_services_on_subheader")
+    : t("home.bluetooth.tracing_off_subheader")
+
   return (
     <View style={style.container}>
-      <View style={style.contentContainer}>
+      <ImageBackground style={style.backgroundImage} source={backgroundImage} />
+      <View style={style.textContainer}>
         <GlobalText style={style.headerText} testID={"home-header"}>
           {headerText}
         </GlobalText>
         <GlobalText style={style.subheaderText} testID={"home-subheader"}>
           {subheaderText}
         </GlobalText>
       </View>
-      {!isEnabledAndAuthorized ? (
-        <Button
-          invert
-          testID={"home-request-permissions-button"}
-          onPress={handleRequestPermission}
-          label={buttonText}
-        />
-      ) : null}
+      <SafeAreaView style={{ flex: 1 }}>
+        <ScrollView style={bottomContainerStyle}>
+          <TouchableOpacity
+            style={style.shareContainer}
+            onPress={handleOnPressShare}
+          >
+            <View style={style.shareImageContainer}>
+              <Image source={Images.HugEmoji} style={style.shareImage} />
+            </View>
+            <View style={style.shareTextContainer}>
+              <GlobalText style={style.shareText}>
+                {t("home.bluetooth.share")}
+              </GlobalText>
+            </View>
+            <View style={style.shareIconContainer}>
+              <SvgXml
+                xml={Icons.Share}
+                width={Iconography.small}
+                height={Iconography.small}
+              />
+            </View>
+          </TouchableOpacity>
+          <View style={style.activationStatusSectionContainer}>
+            <ActivationStatusSection
+              headerText={t("home.bluetooth.bluetooth_header")}
+              isActive={isBluetoothOn}
+              fixAction={handleOnPressBluetooth}
+              testID={"home-bluetooth-status-container"}
+            />
+            <ActivationStatusSection
+              headerText={t("home.bluetooth.proximity_tracing_header")}
+              isActive={isProximityTracingOn}
+              fixAction={handleOnPressProximityTracing}
+              testID={"home-proximity-tracing-status-container"}
+            />
+          </View>
+          <Button
+            onPress={() => navigation.navigate(Stacks.AffectedUserStack)}
+            label={t("home.bluetooth.report_positive_result")}
+            customButtonStyle={style.button}
+            hasRightArrow
+          />
+        </ScrollView>
+      </SafeAreaView>
     </View>
   )
 }
 
+interface ActivationStatusProps {
+  headerText: string
+  isActive: boolean
+  fixAction: () => void
+  testID: string
+}
+
+const ActivationStatusSection: FunctionComponent<ActivationStatusProps> = ({
```

Comment:
Activation is a new domain concept for me. Does this mean 'Things that require permissions to be active?' if so what do you think about `PermissionStatusIndicator` for a name here?

Either way, is `Activation` a word that used used in the business domain? or is this something we are introducing in the codebase?

Reply:
It means a section that displays an active or disabled state to the user, e.g. "Bluetooth is active" or "Proximity tracing is active". IMO `PermissionStatusIndicator` wouldn't work because Bluetooth isn't really a permission. We're just telling the user whether Bluetooth is on or off because EN uses it. Does that make sense?

Reply:
it does, thanks for the explanation!